protoc-gen-grpc-gateway/descriptor/registry.go

```diff
@@ -50,8 +50,12 @@ type Registry struct {
 	// allowRepeatedFieldsInBody permits repeated field in body field path of `google.api.http` annotation option
 	allowRepeatedFieldsInBody bool
 
-	// useJSONNamesForFields if it true Field.GetJsonName() will be used for generating swagger definitions,
-	// otherwise Field.GetName() will be used
+	// useJSONNamesForFields if true json tag name is used for generating fields in swagger definitions,
+	// otherwise original proto name is used. It's helpful for synchronization swagger definition with grpc-gateway
+	// response, if it uses json tags for marshaling.
+	// Pay attention. By default grpc-gataway uses original proto name in responses. If you would like
+	// to use names from json tags change marshaller for gateway with runtime.ServeMuxOption
+	// runtime.WithMarshalerOption(runtime.MIMEWildcard, &runtime.JSONPb{OrigName: false})
```

Comment:
While this is helpful, I don't like the tone of this. How about:
```
// otherwise the original proto name is used. It's helpful for synchronizing the swagger definition with grpc-gateway
// response, if it uses json tags for marshaling.
```

Reply:
done!